assembl/static2/js/app/utils/httpRequestHandler.js

```diff
@@ -1,24 +1,46 @@
+import urljoin from 'url-join';
+import Cookies from 'js-cookie';
+
+import { getCSRFToken } from './csrf';
+import { basePath } from './server';
+
 const convertToURLEncodedString = obj =>
   Object.keys(obj)
     .map(k => `${encodeURIComponent(k)}=${encodeURIComponent(obj[k])}`)
     .join('&');
 const getResponseContentType = xhr =>
   // eslint-disable-line no-unused-vars
   xhr.getResponseHeader('Content-Type').split(';')[0];
+
+const useCSRFProtection = document.getElementById('useCSRFProtection')
```

Comment:
same comment as in `assembl/static2/js/app/client.js`. Since it is the exact same condition as above, we can refactore some code here and import the required utility function here and in `assembl/static2/js/app/client.js` instead, what do you think ?